### PR TITLE
Use PyTest tmpdir fixture instead of manually handling them

### LINKED ISSUE #70: Use the PyTest tmpdir feature instead of manually handling temporary directories through tempfile
There's a nice pytest feature that you can use instead of manually using tempfile: https://docs.pytest.org/en/latest/tmpdir.html.

See also PR #63.

### PR DESCRIPTION
**Pull request by poush**
Fixes #70

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -10,20 +9,20 @@ def test_get_bugs():
     bugs = utils.get_bugs()
     assert(isinstance(bugs, list))
 
+@pytest.fixture(scope='session')
```
Comment (marco-c) [inline tests/test_utils.py]: The tests are not fixtures

File: tests/test_utils.py [hunk 2]
```diff
@@ -10,20 +9,20 @@ def test_get_bugs():
...
+def test_mkdir(tmpdir_factory):
```
Comment (marco-c) [inline tests/test_utils.py]: Use `tmpdir` and the path as a string is `tmpdir.strpath`

**Comment by poush** [comment]
Updated. Please check.

File: tests/test_utils.py [hunk 3]
```diff
@@ -11,19 +10,18 @@ def test_get_bugs():
...
 
-def test_mkdir():
-    d = TemporaryDirectory()
-    direc_path = d.name + "/test"
-    utils.mkdir(direc_path)
-    assert(os.path.isdir(direc_path))
-    utils.mkdir(direc_path)
+def test_mkdir(tmpdir):
+    dir_path = tmpdir.strpath + "/test"
+    utils.mkdir(dir_path)
```
Comment (marco-c) [inline tests/test_utils.py]: Changing the name of the variable is unnecessary and leads to changes everywhere, it's better to leave it as it was

File: tests/test_utils.py [hunk 4]
```diff
@@ -47,12 +45,12 @@ def test_read_labels():
...
 
-def test_write_labels():
+@pytest.fixture(scope='session')
+def test_write_labels(tmpdir):
     label = {1: 1, 2: 2}
-    d = TemporaryDirectory()
-    file_path = d.name + "/test.csv"
-    utils.write_labels(label, file_name=file_path)
-    assert(os.path.exists(file_path))
+    fn = tmpdir.mkdir("data").join("test.csv")
```
Comment (marco-c) [inline tests/test_utils.py]: No need to create a new directory, use tmpdir directly.
Also, don't change the name of the variable

File: tests/test_utils.py [hunk 5]
```diff
@@ -11,19 +10,18 @@ def test_get_bugs():
...
+def test_mkdir(tmpdir):
+    dir_path = tmpdir.strpath + "/test"
+    utils.mkdir(dir_path)
+    assert(os.path.isdir(dir_path))
+    utils.mkdir(dir_path)
 
 
 def test_load_image(tmpdir):
-    d = TemporaryDirectory()
+    fn = tmpdir.mkdir("data").join("Image.jpg")
```
Comment (marco-c) [inline tests/test_utils.py]: No need to create another directory, use tmpdir directly

File: tests/test_utils.py [hunk 6]
```diff
@@ -47,12 +45,12 @@ def test_read_labels():
     assert(isinstance(labels, dict))
 
 
-def test_write_labels():
+@pytest.fixture(scope='session')
```
[the remark this answers is not in the document]
Comment (poush) [reply tests/test_utils.py]: missed it. updated now. Please check.

**Pull request merged**